labs/automated_gateways.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Automated-Gateways
+parent: Labs
+---
+
+# Lab Name
+ Automated-Gateways
+# Short Description
+Automated Gateways is a secure solution which can make Contract-Call interoperatibility happen between Hyperledger Fabrics with minimal efforts. 
+
+# Scope of Lab
+By utilizing three smart contracts as the foundation of authentication and authorization, the Automated Gateways module provides a simple and secure method for Hyperledger fabrics to exchange services and data. When a user integrates this module into their code and deploys three predetermined smart contracts, they can grant access to their services to other Hyperledger fabrics. Moreover, they can access services from smart contracts deployed on other Hyperledger fabrics. Through this method, various Hyperledger fabrics can connect and utilize each other's services.
+
+The module offers users both an API and a CLI. With these tools, users can grant access to their services to other blockchains and invoke services on external blockchains.
+
+This contract-calls interoperability enhances the connectivity of Hyperledger Fabric, reduces the need for reading extensive documentation and protocols from third-party solutions to link different blockchains, and ultimately minimizes service redundancy within the Hyperledger Fabric community.
+
+* This project was developed under the [Hyperledger Mentorship Program](https://wiki.hyperledger.org/display/INTERN/Automated+gateways+through+smart+contracts).
+
+* This project was presented in a [Hyperledger Meetup](https://www.youtube.com/live/zVNNh5OCplM?feature=shared).
+
+
+
+
+# Initial Committers
+- https://github.com/Koosha-E-Khorasani
+
+
+
+# Sponsor
+N/A
+
+** This project is developed and maintained by [TCDT Lab](https://tcdt.ca/) of the University of Manitoba
+
+# Pre-existing repository
+
+- https://github.com/tcdt-lab/Automated-Gateways
```

Comment:
It looks like the commits in this repository do not contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
1. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs

Reply:
I though having my name and email as Author (shown after git log) would cover it. Sorry, my bad!
I will take care of it.
![screenshot](https://github.com/hyperledger-labs/hyperledger-labs.github.io/assets/83594673/18173936-2b75-40eb-ad73-10e6219e1c15)

Reply:
It is resolved @tkuhrt .